test/BF16/Integration/lit.local.cfg

```diff
@@ -19,6 +19,29 @@ def has_support(feature):
 
     return True
 
+def is_arch(target):
+    # Arch detection not working on Windows
+    if sys.platform in ['win32']:
+        return False
+
+    try:
+        cmd = subprocess.Popen(
+            ['uname', '-m'], stdout=subprocess.PIPE)
+    except OSError:
+        return False
+
+    out = cmd.stdout.read().decode('ascii')
+    cmd.wait()
+
+    return target in out
+
+
 # AVX512 and SVE should have BF16 support
 if not has_support('avx512') and not has_support('avx2') and not has_support('sve'):
     config.unsupported = True
+
+# Enable only on x86
+# Other targets may use different VNNI blocking scheme that is not compatible with
+# prepacked shapes in some of the tests
+if not is_arch('x86'):
```

Comment:
Do we have a plan to introduce Arm tests in a follow up PR? Should be just a matter of fixing the blocking factors, no?

Reply:
I think we shouldn't have prepacked integration tests as clearly they're not scalable. That's fine for testing passes but not really runtime behavior.
I'd suggest cleaning up tests in general and reenabling them after.

So in some future modulo other priorities.

Reply:
The problem is that we want to start CI on Arm and this will skip all B16 integration tests on Arm. I recommend we either add some in this PR or in a quick follow up.

Reply:
Let's start spinning the Arm CI with this PR. I'll follow up with test cleanup in the next few days.

Reply:
We'll start Arm CI after we move to libxsmm, the runner is there